INSTALL.EKS.md

```diff
@@ -140,6 +140,7 @@ cat >trust-relationships.json <<EOF
             "system:serviceaccount:kpack:controller",
             "system:serviceaccount:korifi:korifi-api-system-serviceaccount",
             "system:serviceaccount:korifi:korifi-controllers-controller-manager",
+            "system:serviceaccount:korifi:korifi-kpack-image-builder-controller-manager",
```

Comment:
our terraform templates also add the other service accounts:

```
        "system:serviceaccount:kpack:controller",
        "system:serviceaccount:korifi:korifi-api-system-serviceaccount",
        "system:serviceaccount:korifi:korifi-controllers-controller-manager",
        "system:serviceaccount:korifi:korifi-kpack-image-builder-controller-manager",
        "system:serviceaccount:korifi:korifi-statefulset-runner-controller-manager",
        "system:serviceaccount:korifi:korifi-job-task-runner-controller-manager",
        "system:serviceaccount:*:kpack-service-account"
```

I wonder whether they are really needed. @benjaminguttmann-avtq does that change work for you? I.e. are you able push apps?

Reply:
Adding that one line made the push work for me again. Yes

Reply:
Thanks!

We should probably update our terraform scripts to match the documentation then @georgethebeatle

Reply:
I guess we just followed the intertia. When I think about it the only code that attempts to create a repository in the image registry is clearly the kpack image builder. The other serviceaccounts consume the built images. Which makes me wonder if the controller manager needs to be in the list...

Reply:
> Which makes me wonder if the controller manager needs to be in the list...

Good catch, maybe not